web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -931,4 +931,9 @@
     contain: strict;
     scrollbar-width: none;
   }
+
+  #bucket {
```

Comment:
id's are supposed to be unique across the whole document, and we're reusing it in a loop. Maybe this should be a class instead of an id.